ee/codegen/src/generators/node-port.ts

```diff
@@ -3,24 +3,25 @@ import { AstNode } from "@fern-api/python-ast/core/AstNode";
 import { Writer } from "@fern-api/python-ast/core/Writer";
 
 import { WorkflowContext } from "src/context";
-import { GenericNodeContext } from "src/context/node-context/generic-node";
+import { BaseNodeContext } from "src/context/node-context/base";
```

Comment:
love it 👍 

in general, we want to keep progressing from things being just GenericNode -> All Nodes (eg, BaseNode)